packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -68,19 +79,43 @@ export type AgentResponse = {
    * This Specify the teams id which got assigned to the agent.
    */
   teamIds: string[];
+
+  /**
+   * skillProfileId of the agent.
+   */
+  skillProfileId: string;
+
+  /**
+   * siteId of the agent.
+   */
+  siteId: string;
+
+  /**
+   * dbId of the agent.
+   */
+  dbId?: string;
+
+  /**
+   * The default dialled number of the agent.
+   */
+  defaultDialledNumber?: string;
+
+  /**
+   * This Specify the team id which got assigned to the agent.
```

Comment:
shouldn't this be related to supervisor in some way?

Reply:
removed